lib/thor/actions.rb

```diff
@@ -219,7 +219,7 @@ def apply(path, config = {})
 
       contents = if is_uri
         require "open-uri"
-        open(path, "Accept" => "application/x-thor-template", &:read)
+        URI.send(:open, path, "Accept" => "application/x-thor-template", &:read)
```

Comment:
Just curious, is `URI.open(path, ...)` not backwards compatible?

Reply:
There's another part of the codebase that uses the same `send` pattern￼…I think because `open` used to be a private method.￼

Reply:
TIL 👍